Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -964,10 +964,12 @@
   categories:
   - UplinkJob
   conditions:
-    - !type:BuyerJobCondition
+    - !type:BuyerDepartmentCondition
       whitelist:
-        - Scientist
-        - SeniorResearcher
+      - Science
+    - !type:BuyerJobCondition
+      blacklist:
+      - ResearchAssistant # No fun for trainee roles
```

Comment:
just remove this. this seems like an oversight anyway

Reply:
Removed it. Should assistants also get SyndicateSpongeBox?